src/source/PeerConnection/Rtcp.c

```diff
@@ -92,7 +91,6 @@ static STATUS onRtcpReceiverReport(PRtcpPacket pRtcpPacket, PKvsPeerConnection p
     // https://tools.ietf.org/html/rfc3550#section-6.4.2
     if (pRtcpPacket->payloadLength != RTCP_PACKET_RECEIVER_REPORT_MINLEN) {
         // TODO: handle multiple receiver report blocks
-        DLOGS("unhandled packet type RTCP_PACKET_TYPE_RECEIVER_REPORT size %d", pRtcpPacket->payloadLength);
```

Comment:
No need to remove DLOGS

Reply:
Is this actually incorrect/misleading?  Why not keep it as an info level log?

Reply:
It is not required because it does not provide any useful information. Even as an INFO level log, the logs would be flooded with these messages. INFO is useful in one way when customers do not have the luxury to have DEBUG/VERBOSE enabled in production. Considering that and the flooding this specific log generates, it is better to get rid of it.